linalg/src/x86_64_fma.rs

```diff
@@ -96,14 +100,71 @@ fn plug_fma(ops: &mut Ops) {
 fn plug_avx512f(ops: &mut Ops) {
     ops.mmv_f32 = Box::new(|m, _k| match m {
         Some(m) if m < 31 => mmm::avx512_mmm_f32_16x1::mmm(),
-        _ => mmm::avx512_mmm_f32_128x1::mmm(),
+        _ => mmm::avx512_mmm_f32_96x1::mmm(),
```

Comment:
For MMV, we're hitting very low throughput all across the board, regardless of M
My thinking was that according to my benchmarks, lowering 128 to 96 does not cause harm, and it could help with border kernels on matrices that are not multiples of 128